wordpress-mobile/WordPress-iOS
# Reader Saved Posts: replace No Results view

Fixes #9999 

In the Reader > Saved Posts, replace `WPNoResultsView` with `NoResultsViewController`.

The No Results view displayed when there are no Saved Posts.

To test:
- On a site with no saved posts, go to Reader > Saved Posts.
- Verify the view is:
![saved_posts_portrait](https://user-images.githubusercontent.com/1816888/44235626-6a2f7a80-a167-11e8-9c5a-2620f08301ed.png)

![saved_posts_landscape](https://user-images.githubusercontent.com/1816888/44235631-6dc30180-a167-11e8-87b0-701e8c3d7b60.png)
